OMEdit/OMEdit/OMEditGUI/Resources/nls/OMEdit_zh_CN.ts

```diff
@@ -6,60 +6,60 @@
     <message>
         <location filename="../../MainWindow.cpp" line="3791"/>
         <source>About %1</source>
-        <translation type="unfinished"></translation>
+        <translation>关于 %1</translation>
     </message>
     <message>
         <location filename="../../MainWindow.cpp" line="3794"/>
         <source>&lt;h2&gt;%1 - %2&lt;/h2&gt;&lt;b&gt;%3&lt;/b&gt;&lt;br /&gt;&lt;b&gt;Connected to %4&lt;/b&gt;&lt;br /&gt;&lt;br /&gt;Copyright &lt;b&gt;Open Source Modelica Consortium (OSMC)&lt;/b&gt;.&lt;br /&gt;Distributed under OSMC-PL and GPL, see &lt;u&gt;&lt;a href=&quot;http://www.openmodelica.org&quot;&gt;www.openmodelica.org&lt;/a&gt;&lt;/u&gt;.&lt;br /&gt;&lt;br /&gt;Initially developed by &lt;b&gt;Adeel Asghar&lt;/b&gt; and &lt;b&gt;Sonia Tariq&lt;/b&gt; as part of their final master thesis.&lt;br /&gt;&lt;em&gt;Compiled without 3D animation support&lt;/em&gt;.&lt;br /&gt;&lt;br /&gt;&lt;b&gt;Contributors:&lt;/b&gt;&lt;ul&gt;&lt;li&gt;Adeel Asghar - &lt;u&gt;&lt;a href=&quot;mailto:adeel.asghar@liu.se&quot;&gt;adeel.asghar@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Sonia Tariq&lt;/li&gt;&lt;li&gt;Martin Sjölund - &lt;u&gt;&lt;a href=&quot;mailto:martin.sjolund@liu.se&quot;&gt;martin.sjolund@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Alachew Shitahun - &lt;u&gt;&lt;a href=&quot;mailto:alachew.mengist@liu.se&quot;&gt;alachew.mengist@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Jan Kokert - &lt;u&gt;&lt;a href=&quot;mailto:jan.kokert@imtek.uni-freiburg.de&quot;&gt;jan.kokert@imtek.uni-freiburg.de&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Dr. Henning Kiel - &lt;u&gt;&lt;a href=&quot;mailto:henning.kiel@w-hs.de&quot;&gt;henning.kiel@w-hs.de&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Haris Kapidzic&lt;/li&gt;&lt;li&gt;Abhinn Kothari&lt;/li&gt;&lt;li&gt;Lennart Ochel - &lt;u&gt;&lt;a href=&quot;mailto:lennart.ochel@liu.se&quot;&gt;lennart.ochel@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Volker Waurich - &lt;u&gt;&lt;a href=&quot;mailto:volker.waurich@tu-dresden.de&quot;&gt;volker.waurich@tu-dresden.de&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Rüdiger Franke&lt;/li&gt;&lt;li&gt;Martin Flehmig&lt;/li&gt;&lt;li&gt;Robert Braun - &lt;u&gt;&lt;a href=&quot;mailto:robert.braun@liu.se&quot;&gt;robert.braun@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Per Östlund - &lt;u&gt;&lt;a href=&quot;mailto:per.ostlund@liu.se&quot;&gt;per.ostlund@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;li&gt;Dietmar Winkler&lt;/li&gt;&lt;li&gt;Anatoly Severin&lt;li&gt;&lt;li&gt;Adrian Pop - &lt;u&gt;&lt;a href=&quot;mailto:adrian.pop@liu.se&quot;&gt;adrian.pop@liu.se&lt;/a&gt;&lt;/u&gt;&lt;/li&gt;&lt;/ul&gt;</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Why is this empty instead of unfinished?

Reply:
Nice catch. Maybe this should be unfinished Or I can try google translator to find a proper Chinese translation of it.